systemvm/patches/debian/config/opt/cloud/bin/cs/CsAddress.py

```diff
@@ -369,18 +369,18 @@ def fw_router(self):
 
         if self.get_type() in ["public"]:
             self.fw.append(["mangle", "front",
-                            "-A PREROUTING " +
+                            "-I PREROUTING " +
```

Comment:
FYI This is not needed, since there is "front" it will already become -I. See CsNetfilter.py